Resources/Prototypes/DeltaV/Entities/Objects/Devices/station_beacon.yml

```
@@ -1,113 +1,113 @@
 #Command
 - type: entity
-  parent: DefaultStationBeaconCommand
+  parent: DefaultStationBeacon
   id: DefaultStationBeaconConferenceRoom
   suffix: Conference Room
   components:
   - type: NavMapBeacon
-    text: station-beacon-conference-room
+    defaultText: station-beacon-conference-room
 
 #Medical
```

yamllint

[warning] missing starting space in comment

(comments)